src/scss/includes/components/contextMenu.scss

```diff
@@ -0,0 +1,44 @@
+.contextMenu {
+  position: relative;
+
+  &-button {
+    height: 30px;
+    width: 30px;
+    display: block;
+    border-radius: 50%;
+    text-align: center;
+    line-height: 30px;
+    color: $primary_clear;
+
+    &:hover {
+      background: $separator_color;
+      color: $primary_alternative_text;
+      cursor: pointer;
+    }
+  }
+
+  &-menu {
+    position: absolute;
+    right: 36px;
+    top: -9px;
+    padding: 9px;
+    background: $background;
+    color: $primary_text;
+    font-size: 13px;
+    border-radius: 3px;
+    box-shadow: 0 2px 12px 0 rgba(0, 0, 0, 0.2);
+  }
```

Comment:
With this new positioning, the last item may collide with the Masq footer, (solvable with z-index ?):
![image](https://user-images.githubusercontent.com/4726554/67017700-6e07b580-f0fa-11e9-969b-bda88595cceb.png)

Also visible at the bottom of the scrollable panel:
![image](https://user-images.githubusercontent.com/4726554/67017968-cf2f8900-f0fa-11e9-8811-83f806257239.png)